client/components/button/README.md

```diff
@@ -19,6 +19,7 @@ render: function() {
 
 * `compact`: (bool) whether the button is compact or not.
 * `primary`: (bool) whether the button is styled as a primary button.
+* `secondary`: (bool) whether the button is styled as a secondary button.
```

Comment:
^ The corollary I left out from the first paragraph is that there will probably not be consensus to add another prop to Button.